mii/models/score/score_template.py

```diff
@@ -16,24 +16,41 @@
 
 def init():
     model_path = mii.utils.full_model_path(configs[mii.constants.MODEL_PATH_KEY])
-
+    deployment_tag = configs[mii.constants.DEPLOYMENT_TAG_KEY]
+    deployments = []
+    lb_enabled = False
+    for deployment in configs[mii.constants.DEPLOYMENTS_KEY].values():
+        if deployment[mii.constants.DEPLOYED_KEY]:
+            lb_enabled = True
+            print(deployment)
+            continue
+        data = {
+            'deployment_name': deployment[mii.constants.DEPLOYMENT_NAME_KEY],
+            'task': deployment[mii.constants.TASK_NAME_KEY],
+            'model': deployment[mii.constants.MODEL_NAME_KEY],
+            'enable_deepspeed': deployment[mii.constants.ENABLE_DEEPSPEED_KEY],
+            'enable_zero': deployment[mii.constants.ENABLE_DEEPSPEED_ZERO_KEY],
+            'GPU_index_map': deployment[mii.constants.GPU_INDEX_KEY],
+            'mii_config': deployment[mii.constants.MII_CONFIGS_KEY],
+            'ds_config': deployment[mii.constants.DEEPSPEED_CONFIG_KEY],
+            'version': 1
```

Comment:
If we made the `mii.constants.*` values and the `DeploymentConfig` field names match (or add field aliases) we could just pass the entire `deployment` dict to `DeploymentConfig` rather than be extra verbose with the packing and unpacking of values.